ddtrace/tracer/span.go

```diff
@@ -114,8 +114,6 @@ func (s *span) SetTag(key string, value interface{}) {
 		s.setMetric(key, v)
 		return
 	}
-	// not numeric, not a string and not an error, the likelihood of this
```

Comment:
Why was this removed? Does it no longer apply? Seems like it does.

Reply:
Since with this change large integers go through here, I think this comment becomes misleading. The likelihood is not close to zero anymore. I can leave it or modify it if you think it's still valuable.